packages/flinks-token-exchange/.env.local.example

```diff
@@ -0,0 +1,10 @@
+# Dwolla Environment and Client Key and Secret
+# https://dashboard-sandbox.dwolla.com/applications-legacy
+DWOLLA_ENV=sandbox
+DWOLLA_KEY=
+DWOLLA_SECRET=
+
+# Flinks Instance, Customer ID, and API Secret
```

Comment:
```suggestion
# Flinks Instance, Customer ID, and API Secret
# Contact Flinks (https://flinks.com/contact/sales/) for more information on how to obtained these values
```

Reply:
@joeyhage Something like this work? Mainly to have the user get these values from Flinks, since it doesn't look like they're posted anywhere online?